### ISSUE TITLE
[Exploit] Infinite throughput on Poles

### ISSUE THREAD

**Issue by OmegaHaxors** [Electrical-Age/ElectricalAge#759]
Transmission lines can handle infinite current on the ground wire which allows users to bypass wires completely and have entire networks of equipment run off of one extremely overamped line.

![2017-08-20_15 25 16](https://user-images.githubusercontent.com/16057503/29499034-dfaa5810-85bc-11e7-9223-773bb93d7069.png)

Related: The converter poles will hum when large currents are passed through the ground wire, even if there is no energy passing into the overhead line.

**Comment by AMIDIBOSS** [comment]
This will be usefull...

**Comment by OmegaHaxors** [comment]
I mean sure if you want to put 2.1 Gigawatts through a bunch of relays and using poles to turn corners that's fine, but I did put the tag [Exploit] there for a reason. Have a little fun with it in Creative mode before it gets patched.

**Comment by AMIDIBOSS** [comment]
IF it gets patched...

**Comment by AMIDIBOSS** [comment]
you see, some components don't have temp. watchdog nor current watchdog
Poles have them only between eachother, but not on the ground. Just like with converters long ago...

**Comment by pooing1234** [comment]
YES! I can avoid using double height lines to power my city, farming facilities, and factories!

**Comment by AMIDIBOSS** [comment]
You see! Problem Solved! No need for t2!

**Comment by OmegaHaxors** [comment]
Except this only works from ground wire to ground wire. If you try to send that power to the line itself, the current limit of the sky wire will cause the entire thing to come tumbling down.

**Comment by pooing1234** [comment]
@OmegaHaxors Yep. I nearly blew half my grid up. Thew!

**Comment by pooing1234** [comment]
@OmegaHaxors Yep, nearly blew half of my grid up. Gonna have to use this method. http://imgur.com/a/1Xoid

**Comment by jrddunbr** [comment]
Turns out it also works with other stuff too:

![2017-12-29_20 07 20](https://user-images.githubusercontent.com/14130505/34450322-b82ca024-eccb-11e7-8e4d-4ae026f61121.png)

**Comment by OmegaHaxors** [comment]
Interesting. I always knew the inductor and relay were unlimited but the hub and detector I honestly assumed would have had a limit considering they have slightly higher resistances.
